src/server/schema/image.schema.ts

```diff
@@ -297,6 +297,13 @@ export const getInfiniteImagesSchema = baseQuerySchema
     pending: z.boolean().optional(),
     tools: z.number().array().optional(),
     techniques: z.number().array().optional(),
+    baseModels: z
+      .union([z.enum(constants.baseModels), z.enum(constants.baseModels).array()])
+      .optional()
+      .transform((rel) => {
+        if (!rel) return undefined;
+        return Array.isArray(rel) ? rel : [rel];
+      }),
```

Comment:
If we are going to transform it to an array anyways, lets just accept enum array only instead of the union
```suggestion
    baseModels: z.enum(constants.baseModels).array().optional(),
```